.ebextensions/01_create_apache_conf.config

```diff
@@ -126,7 +126,6 @@ files:
         SSLSessionTickets     Off
 
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains; preload"
-        Header always set X-Frame-Options DENY
```

Comment:
Note that this had to be taken out since it prevented using the tools within an iframe. The fact that it's been hanging out in our ssl.conf for months implies that up until now the entire ssl.conf was being ignored.